bigbluebutton-html5/imports/ui/components/video-dock/component.jsx

```diff
@@ -287,6 +287,7 @@ class VideoDock extends Component {
       if (that.myId == id) {
         that.notifyError(intl.formatMessage(intlMessages.sharingError));
         that.unshareWebcam();
+        that.stop(id);
```

Comment:
Please bring `that.stop(id)` outside of the condition block. Right now it's both in the `if` and `else`

Reply:
Yeah and I even think you can just move add a `this.stop(myId)` inside the `unshareWebcam` method and it work without any additional code.